lgc/patch/PatchCopyShader.cpp

```diff
@@ -218,8 +227,6 @@ bool PatchCopyShader::runImpl(Module &module, PipelineShadersResult &pipelineSha
 
   if (m_pipelineState->isGsOnChip())
     m_lds = Patch::getLdsVariable(m_pipelineState, &module);
-  else
-    m_gsVsRingBufDesc = loadGsVsRingBufferDescriptor(builder);
```

Comment:
m_gsVsRingBufDesc is no longer used, and I get this warning on clang:

```
lgc/include/lgc/patch/PatchCopyShader.h:52:16: warning: private field 'm_gsVsRingBufDesc' is not used [-Wunused-private-field]
  llvm::Value *m_gsVsRingBufDesc = nullptr; // Descriptor for GS-VS ring
               ^
```

Reply:
Fixed by https://github.com/GPUOpen-Drivers/llpc/pull/2525